fundingmanager.go

```diff
@@ -43,6 +41,13 @@ const (
 	// TODO(roasbeef): add command line param to modify
 	maxFundingAmount = btcutil.Amount(1 << 24)
 
+	// minRemoteDelay and maxRemoteDelay is the extremes of the CSV delay
+	// we will require the remote to use for its commitment transaction.
+	// The actual delay we will require will be somewhere between these
+	// values, depending on channel size.
+	minRemoteDelay = 4
+	maxRemoteDelay = 144
```

Comment:
Granted that the current max channel size is ~$3k...the upper limit I suggested may bit a bit much